app/views/menu/_menu_fandoms.html.erb

```diff
@@ -1,9 +1,9 @@
 <ul class="menu" role="menu">
-  <li><%= span_if_current ts('All Fandoms', key: 'header.fandom'), media_path %></li>
+  <li><%= link_to ts('All Fandoms', key: 'header.fandom'), media_path %></li>
   <% cache "menu-fandoms-version2", skip_digest: true do %>
```

Comment:
Could you bump the version number on this to `menu-fandoms-version3`? That way, the cache will update for everyone when we deploy.